packages/mobile/navigation/AppNavigator.tsx

```diff
@@ -2,49 +2,84 @@ import React from "react";
 import HomeScreen from "../screens/Home";
 import { createDrawerNavigator } from "@react-navigation/drawer";
 import { createStackNavigator } from "@react-navigation/stack";
-
-import {
-  Image,
-  ScrollView,
-  StyleSheet,
-  Text,
-  TouchableOpacity,
-  View
-} from "react-native";
-import { Button } from "react-native-paper";
+import { IconButton } from "react-native-paper";
+import { View } from "react-native";
+import ArticleContent from "../components/ArticleContent";
 
 const Drawer = createDrawerNavigator();
 const Stack = createStackNavigator();
 
-export default AppNavigator = () => (
-  <Drawer.Navigator drawerType="front">
-    <Drawer.Screen name="Welcome" component={StackNavigator} />
-  </Drawer.Navigator>
-);
+export default function AppNavigator() {
+  return (
+    <Drawer.Navigator drawerType="front">
+      <Drawer.Screen name="Welcome" component={StackNavigator} />
+    </Drawer.Navigator>
+  );
+}
 
-const StackNavigator = ({ navigation, route }) => (
+const StackNavigator = ({
+  navigation,
+  route,
+}: {
+  navigation: any;
+  route: any;
+}) => (
   <Stack.Navigator
     initialRouteName="Home"
     mode="modal"
     headerMode="float"
     screenOptions={{
       headerStyle: { backgroundColor: "#FFC200" },
       headerLeft: () => (
-        <Button
+        <IconButton
           style={{ minWidth: 0, left: 4 }}
           icon="menu"
           onPress={() => navigation.toggleDrawer()}
         />
       ),
-      headerTitleContainerStyle: { left: 44 }
+      headerTitleContainerStyle: {
+        left: 4,
```

Comment:
This is intended to be 44, why the change?

Reply:
Why 44? To center the title?